src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/NotIngressFlowSegmentCommand.java

```diff
@@ -15,6 +15,8 @@
 
 package org.openkilda.floodlight.command.flow;
 
+import static org.openkilda.floodlight.switchmanager.SwitchManager.VXLAN_UDP_DST;
```

Comment:
Looks weird - accessing a "service" class (which is higher in the hierarchy) from a shared "action" class (which is lower).